apps/services/auth/public-api/infra/auth-public-api.ts

```diff
@@ -27,39 +39,25 @@ export const serviceSetup = (): ServiceBuilder<'services-auth-public-api'> => {
         staging: 'IS-TEST/GOV/6503760649/SKRA-Protected/Einstaklingar-v1',
         prod: 'IS/GOV/6503760649/SKRA-Protected/Einstaklingar-v1',
       },
-      XROAD_NATIONAL_REGISTRY_REDIS_NODES: {
-        dev: json([
-          'clustercfg.general-redis-cluster-group.5fzau3.euw1.cache.amazonaws.com:6379',
-        ]),
-        staging: json([
-          'clustercfg.general-redis-cluster-group.ab9ckb.euw1.cache.amazonaws.com:6379',
-        ]),
-        prod: json([
-          'clustercfg.general-redis-cluster-group.dnugi2.euw1.cache.amazonaws.com:6379',
-        ]),
-      },
-      XROAD_RSK_PROCURING_REDIS_NODES: {
-        dev: json([
-          'clustercfg.general-redis-cluster-group.5fzau3.euw1.cache.amazonaws.com:6379',
-        ]),
-        staging: json([
-          'clustercfg.general-redis-cluster-group.ab9ckb.euw1.cache.amazonaws.com:6379',
-        ]),
-        prod: json([
-          'clustercfg.general-redis-cluster-group.dnugi2.euw1.cache.amazonaws.com:6379',
-        ]),
-      },
+      XROAD_NATIONAL_REGISTRY_REDIS_NODES: REDIS_NODE_CONFIG,
+      XROAD_RSK_PROCURING_REDIS_NODES: REDIS_NODE_CONFIG,
```

Comment:
Why not use the `.redis()` directive to get Redis access and use that instead?

Reply:
Do you then reference `REDIS_URL_NODE_01` to get the value in the configs?

Reply:
I'll look into this when I address the // TODO comments in another PR after testing on dev